src/ApiService/ApiService/onefuzzlib/Containers.cs

```diff
@@ -185,7 +254,7 @@ public class Containers : IContainers {
 
         var sasBuilder = new BlobSasBuilder(permissions, endTime) {
             StartsOn = startTime,
-            BlobContainerName = container.ContainerName,
+            BlobContainerName = _config.OneFuzzStoragePrefix + container.ContainerName,
```

Comment:
this will be a source of hard to debug bugs forgetting to add prefix to container name. 

Can enforce that container name always requires prefix ?

Reply:
At the moment we won't have any problems since production will always use an empty prefix. I'll think about a way to require that this is provided…

Reply:
Discussed that we will address this later, as it won't affect production, only testing.